supply_energy.md

```diff
@@ -0,0 +1,291 @@
+
+## Equations 
+The equations that determine energy supply are described here.
+
+### Renewable resource supply curves
+
+The specific supply curve in each region for wind and solar is assigned three parameters, detailed in the following equation:
+
+$$
+Q = maxSubResource * \frac{P^{CurveExponent} }{ ( {MidPrice^{CurveExponent} + P^{CurveExponent} ) } }
+$$
+
+Where Q refers to the quantity of electricity produced, P the price, and the remaining parameters are exogenous, with the names in the XML input files corresponding to the names in the equation above. maxSubResource indicates the maximum quantity of renewable energy that could be produced at any price, curve-exponent is a shape parameter, and mid-price indicates the price at which 50% of the maximum available resource is produced.
+
+See `annualsupply` in [smooth_renewable_subresource.cpp](https://github.com/JGCRI/gcam-core/blob/master/cvs/objects/resources/source/smooth_renewable_subresource.cpp).
+
+## Policy options 
+This section summarizes some of the energy-based policy options available in GCAM. 
+
+### Energy production policies
+
+Users can include [energy production policies](policies.html#energy-production-policies), which can constrain energy production or create incentives to increase or decrease production. 
+
+### Energy intensity standards
+
+Users can also include [energy intensity standards](policies_examples.html#res), such as a renewable energy standard.
+
+## Insights and intuition
+
+### Paper/Topic #1
+
+{One paragraph summary of a key insight from one or more papers}
+
+## IAMC Reference Card
+
+### Energy technology substitution
+
+Energy technology choice
+- [ ] No discrete technology choices
+- [X] Logit choice model
+- [ ] Production function
+- [ ] Linear choice (lowest cost)
+- [ ] Lowest cost with adjustment penalties
+
+Energy technology substitutability
+- [ ] Mostly high substitutability
+- [ ] Mostly low substitutability
+- [X] Mixed high and low substitutability
+
+Energy technology deployment
+- [ ] Expansion and decline constraints
+- [X] System integration constraints
+
+### Energy
+
+Electricity technologies
+- [X] Coal w/o CCS
+- [X] Coal w/ CCS
+- [X] Gas w/o CCS
+- [X] Gas w/ CCS
+- [X] Oil w/o CCS
+- [X] Oil w/ CCS
+- [X] Bioenergy w/o CCS
+- [X] Bioenergy w/ CCS
+- [X] Geothermal power
+- [X] Nuclear power
+- [X] Solar power
+- [X] Solar power-central PV
+- [X] Solar power-distributed PV
+- [X] Solar power-CSP
+- [X] Wind power
+- [X] Wind power-onshore
+- [X] Wind power-offshore
+- [X] Hydroelectric power
+- [ ] Ocean power
+
+Hydrogen production
+- [X] Coal to hydrogen w/o CCS
+- [X] Coal to hydrogen w/ CCS
+- [X] Natural gas to hydrogen w/o CCS
+- [X] Natural gas to hydrogen w/ CCS
+- [X] Oil to hydrogen w/o CCS
+- [X] Oil to hydrogen w/ CCS
+- [X] Biomass to hydrogen w/o CCS
+- [X] Biomass to hydrogen w/ CCS
+- [X] Nuclear thermochemical hydrogen
+- [X] Solar thermochemical hydrogen
```

Comment:
GCAM's solar and wind hydrogen techs are electrolysis with dedicated renewable energy inputs.  I think "solar thermochemical hydrogen" is more like nuclear thermochemical hydrogen (thermal splitting) but with CSP (solar heat), so this one should be unchecked.